build.gradle

```diff
@@ -63,7 +63,7 @@ repositories {
 }
 
 dependencies {
-    compile fileTree(include: ['*.jar'], dir: 'libs')
+    //compile fileTree(include: ['*.jar'], dir: 'libs')
```

Comment:
Does this commented line has any significance? If so, please add comment that documents it. If not, please delete it.